src/Tasks/Microsoft.NET.Build.Tasks/PrepareForReadyToRunCompilation.cs

```diff
@@ -49,10 +49,13 @@ public class PrepareForReadyToRunCompilation : TaskBase
         [Output]
         public ITaskItem[] ReadyToRunFilesToPublish => _r2rFiles.ToArray();
 
+        [Output]
+        public ITaskItem[] ReadyToRunFilesToReference => _r2rReferences.ToArray();
```

Comment:
```suggestion
        public ITaskItem[] ReadyToRunAssembliesToReference => _r2rReferences.ToArray();
```

Reply:
This should fix the failing tests. Running tests on my dev box now...

Reply:
Thanks, that was a last minute change and I haven't gotten back to this PR since I returned from vacation yesterday.